src/utils/similarWord.js

```diff
@@ -0,0 +1,92 @@
+const mnemonic = require('bitcore-mnemonic');
```

Comment:
Why `require` instead of `import`? I suspect there is a reason but don't know what it is. A comment would be nice.